package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@a2lix/symfony-collection",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
This is not a bugfix, it's a new feature. It should go in a minor version, not in a patch version. So, 0.5.0.

Reply:
Done